.github/workflows/settings.xml

```diff
@@ -0,0 +1,18 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+						  https://maven.apache.org/xsd/settings-1.0.0.xsd">
+   <servers>
+      <server>
+         <id>github-datawave</id>
+         <configuration>
+            <httpHeaders>
+               <property>
+                  <name>Authorization</name>
+                  <value>Bearer ${env.GITHUB_TOKEN}</value>
```

Comment:
Different then other datawave repos. I think it is better to use a GITHUB_TOKEN token instead of a PAT because it is onetime use per workflow run.
Open to discussions on this. Maybe a change that we can apply to the other repos if we like it